sdk/monitor/azure-monitor-ingestion/tests/perf_tests/README.md

```diff
@@ -0,0 +1,53 @@
+# Monitor Ingestion performance tests
+
+In order to run the performance tests, the `azure-devtools` package must be installed. This is done as part of the `dev_requirements`.
+Start by creating a new virtual environment for your perf tests. This will need to be a Python 3 environment, preferably >=3.7.
+
+### Setup for test resources
+
+These tests will run against a pre-configured Log Analytics workspace. The `test-resources.json` file can be used to provision the needed resources. The following environment variables will need to be set for the tests to access the live resources:
+```
+AZURE_MONITOR_DCR_ID=<The ID of the data collection rule>
+AZURE_MONITOR_DCE=<The data collection endpoint to upload logs to>
```

Comment:
Nit - i believe we use different names than here in samples. I would suggest using the same, so it's easy to set up env variables

Reply:
Currently using environment variables output from deploying the test-resources.json. I will look into making environment variables consistent as I start adjusting the tests in the near term.